docs/browser.md

```diff
@@ -60,7 +62,7 @@ This syntax doesn't necessarily work in the browser, but it can be used when bun
 ### Worker
 
 ```js
-importScripts("https://unpkg.com/prettier@1.13.0/standalone.js");
-importScripts("https://unpkg.com/prettier@1.13.0/parser-graphql.js");
+importScripts("https://unpkg.com/prettier@1.18.2/standalone.js");
+importScripts("https://unpkg.com/prettier@1.18.2/parser-graphql.js");
```

Comment:
Maybe `unpkg` has alias like `stable`/`latest` to avoid using versions here, it is not good update them on each new release

Reply:
Simply omit the version number, if you want latest.

Reply:
Maybe we can create issue in `unpkg` and support alias `latest` as latest stable version?

Reply:
Like a wrote, they already have such an URL, but as it is dynamic, it can't be cached and is implemented as a redirect.

Reply:
yes, `https://unpkg.com/prettier/standalone.js` is the latest. but we should suggest people to pin version

Reply:
@thorn0 i was misleading, i mean support latest without redirect :smile:

Reply:
jsdelivr has this feature? forgot

Reply:
That would miss the point of CDN, which is to provide URLs that never change and can be cached forever.

Reply:
@fisker Doesn't Docusaurus have a feature for substitutions like that?

Reply:
> If you want users to be able to use the latest version when you cut a new release, the best policy is to put the version number in the URL directly in your installation instructions. This will also load more quickly because we won't have to resolve the latest version and redirect them.

-- https://unpkg.com/#cache-behavior

Reply:
I was going to put a placeholder there on first thought, then I think may confuse people read this `.md`  directly

Reply:
How about processing all the links to `unpkg.com/prettier`, without relying on `previousVersionOnMaster`?

Reply:
> Anyway, using `previousVersionOnMaster` looks fragile.

do you prefer 

```js
/https:\/\/unpkg\.com\/prettier@(.*?)\//g
```
?

Reply:
I was using this at first, then I thought `previousVersionOnMaster` should be safer

Reply:
I'd prefer this, yes.

Reply:
I can change it, one sec

Reply:
remove the protocol, we may won't strip it some day.

... or, should I strip it now?

Reply:
Let's remove it while you are at it.

Reply:
@evilebottnawi  `https://unpkg.com/prettier@1.13.0/standalone.js` or `//unpkg.com/prettier@1.13.0/standalone.js` ?

Reply:
`https`. There’s never a reason to use anything else, is it?

Reply:
ok , ready to merge